packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -29,8 +30,10 @@ const defaultProps = {
 
 const DetailList = ({ title, children, ...customProps }) => {
   let titleContent;
+  const level = useContext(HeadingLevelContext);
+  const HeaderLevel = `h${level}`;
   if (title) {
-    titleContent = (<div className={cx('title')}>{title}</div>);
+    titleContent = (<HeaderLevel className={cx('title')}>{title}</HeaderLevel>);
```

Comment:
I think we need to add the `id` tag so we can tie this to the `<section>`
https://github.com/cerner/terra-clinical/pull/861/files#r1218574864

cc: @scottwilmarth  / @brooksienoodlesoup

Reply:
@kolkheang - Will this change be done in the new Jira or is that something different and this needs to be fixed as part of this story?

Reply:
@kolkheang  we'll add the Title as a new Jira.